projects/angular-showcase/src/app/features/business-component-viewer/business-component-viewer.module.ts

```diff
@@ -0,0 +1,19 @@
+import { PortalModule } from '@angular/cdk/portal';
```

Comment:
I'm not quite content with the (almost) complete duplication of the public-component-viewer.
If possible, extract common functionality into a base class or inherit where possible from the public variant.

Reply:
@kyubisation Tried to separate it into a 'core' part. Hope that's what you were looking for 😅

Reply:
Sorry, I might have led you astray a bit. You separated it more than I intended it to be. 😃 
But this is absolutely fine. Thank you very much.